CoreConversationView/src/au/gov/asd/tac/constellation/views/conversationview/resources/conversation.css

```diff
@@ -8,7 +8,7 @@
 
 
 .text-area .scroll-bar {
-    -fx-skin: ""; /* com.sun.javafx.scene.control.skin.ScrollBarSkin"; */
+    -fx-skin: ""; /* javafx.scene.control.skin.ScrollBarSkin"; */
```

Comment:
any idea why we're not using this one?

Reply:
I wondered the same thing but assumed it was causing a graphics issue. I plan to clean this all up in #308.